artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -365,8 +365,6 @@ public void setUp() throws Exception {
       InVMRegistry.instance.clear();
 
       // checkFreePort(TransportConstants.DEFAULT_PORT);
-
-      logAndSystemOut("#test " + getName());
```

Comment:
This one I actually think it's important. When looking at failures this will correlate logs with the test. Not having this will make it harder to find what tests are failing because of a bad logging.

Reply:
This bit of information should already be logged by the "watch" rule.